### ISSUE TITLE
floordate mutates the Date object passed

### ISSUE THREAD

**Issue by derhuerst** [brycebaril/node-floordate#2]
``` js
const f = require('floordate')

const x = new Date()
// 2016-08-13T06:19:19.183Z
f(x, 'day')
// 2016-08-12T22:00:00.000Z
x
// 016-08-12T22:00:00.000Z   <-- here!
```

**Comment by derhuerst** [comment]
bump.

**Comment by brycebaril** [comment]
Fixed and released as v3.0.0 (semver major since I don't know if anyone was relying on the mutation)

**Comment by derhuerst** [comment]
> semver major since I don't know if anyone was relying on the mutation

That would be pretty sad! 😁
